docs/release.md

```diff
@@ -0,0 +1,55 @@
+# Status AddOn Relesea process 
+
+The Status AddOn release process is based on [GoReleaser](https://goreleaser.com).
+GoReleaser is configured to automatically create a new release by pushing new tags matching the
+pattern 'v*' and build and publish to ghcr.io the packages for the 
+[container image](https://github.com/kubestellar/ocm-status-addon/pkgs/container/ocm-status-addon)
+and the [helm chart](https://github.com/kubestellar/ocm-status-addon/pkgs/container/ocm-status-addon-chart) 
```

Comment:
But GoReleaser is not doing the Helm chart part. There is one GitHub workflow that both (a) invokes GoReleaser to build and publish the container image and (b) (not using GoReleaser) customizes and publishes the Helm chart.

Reply:
Correct. Your suggested rewrite of the paragraph clarifies that, and I also change at the start to say that release process is based on GitHub Actions - and later you say about the workflow and go releaser and helm chart in the workflow.